psi4/src/psi4/cc/cchbar/globals.h

```diff
@@ -26,6 +26,9 @@
  * @END LICENSE
  */
 
+#ifndef PSI4_CCHBAR_GLOBALS_H
+#define PSI4_CCHBAR_GLOBALS_H
```

Comment:
do we have a pattern for when gets the `psi4` prefix? these are the only ones I see:

```
../psi4/include/psi4/pybind11.h:#ifndef PSI4_CORE_PYBIND11_H_H
../psi4/src/psi4/dfep2/dfep2.h:#ifndef PSI4_GUARD_EPMP2_H
../psi4/src/psi4/libfilesystem/path.h:#ifndef PSI4_CORE_PATH_H
```

Reply:
I do prefer using just `#pragma once` at the top, easier to settle on a name for the `define` when there is not name to provide.

Reply:
`#pragma once` discussion for anyone else who wasn't aware of an alternative to header guards https://en.wikipedia.org/wiki/Pragma_once . compilers we're interested in (except cuda) support it.